### DISCUSSION TITLE
Making a .deb file for easy installation for Linux Mint and other distributions of linux.

### DISCUSSION THREAD

**Discussion by rjsmith899** [navidrome/navidrome#3088] [Ideas]
Making a .deb file for easy installation for Linux Mint and other distributions of linux.

**Comment by deluan** [comment]
Building and maintaning packages for various OSes and distributions is a bit out of the scope of the project, but the community has been awesome and there are packages for various distros out there: https://www.navidrome.org/docs/installation/packages/

If anyone decides to create a .deb package, let me known and I can add to that page to make it easier for new users to find about it.
